src/components/WebexAvatar/WebexAvatar.jsx

```diff
@@ -24,18 +34,48 @@ export default function WebexAvatar({
 }) {
   const {avatar, displayName, status} = usePerson(personID);
   const type = displayStatus ? status : null;
-
+  const iconName = `avatar-status-${status}`;
```

Comment:
I think we should leave the icon names as they are in figma, and create a map from status names to icon names. The icons are not exclusive to avatar usage.

The status class names can simply be computed based on status, eg:
```js
const statusClassName = `status-${status}`;
```